src/module/actions/saveAction.js

```diff
@@ -67,7 +67,7 @@ export function saveAction (api, isCollection, moduleName, defaultQuery = {}) {
             })
         }
       ).then(({ data, status }) => {
-        if (status === 204) {
+        if (status === 204 || status === 200) {
```

Comment:
This would effectively never apply changed data from the server. In this case, on `status === 200`, data should be checked if it is "empty". However, it can be less empty than just `null`, as per Spec, empty responses say nothing about the contents of the `meta` field.

Reply:
fixed in https://github.com/eFrane/vuex-json-api/pull/123/commits/d961426d0c2f5abeeeb7bc25edb54200c050656e